public/docs/ts/_cache/tutorial/toh-pt6.jade

```diff
@@ -183,173 +209,141 @@ block get-heroes-details
+  The `<li>` element should now look like this:
 
-+makeExcerpt('app/hero-detail.component.html', 'save')
++makeExcerpt('app/heroes.component.html', 'li-element')
 
 :marked
-  ### Add/Delete in the *HeroesComponent*
+  In addition to calling the component's `delete` method, the delete button
+  click handling code stops the propagation of the click event &mdash; we
+  don't want the `<li>` click handler to be triggered because that would
+  select the hero that we are going to delete!
 
-  We'll be reporting propagated HTTP errors, let's start by adding the following
-  field to the `HeroesComponent` class:
+  The logic of the `delete` handler is a bit trickier:
 
-+makeExcerpt('app/heroes.component.ts', 'error', '')
++makeExcerpt('app/heroes.component.ts', 'delete')
 
 :marked
-  The user can *add* a new hero by clicking a button and entering a name.
-
-block add-new-hero-via-detail-comp
-  :marked
-    When the user clicks the *Add New Hero* button, we display the `HeroDetailComponent`.
-    We aren't navigating to the component so it won't receive a hero `id`;
-    as we noted above, that is the component's cue to create and present an empty hero.
+  Of course, we delegate hero deletion to the hero service, but the component
+  is still responsible for updating the display: it removes the deleted hero
+  from the !{_array} and resets the selected hero if necessary.
 
-- var _below = _docsFor == 'dart' ? 'before' : 'below';
 :marked
-  Add the following to the heroes component HTML, just !{_below} the hero list (`<ul class="heroes">...</ul>`).
-+makeExcerpt('app/heroes.component.html', 'add-and-error')
-:marked
-  The first line will display an error message if there is any. The remaining HTML is for adding heroes.
+  We want our delete button to be placed at the far right of the hero entry.
+  This extra CSS accomplishes that:
 
-  The user can *delete* an existing hero by clicking a delete button next to the hero's name.
-  Add the following to the heroes component HTML right after the hero name in the repeated `<li>` tag:
-+makeExcerpt('app/heroes.component.html', 'delete')
-:marked
-  Add the following to the bottom of the `HeroesComponent` CSS file:
 +makeExcerpt('app/heroes.component.css', 'additions')
-:marked
-  Now let's fix-up the `HeroesComponent` to support the *add* and *delete* actions used in the template.
-  Let's start with *add*.
 
-  Implement the click handler for the *Add New Hero* button.
-
-+makeExcerpt('app/heroes.component.ts', 'addHero')
+:marked
+  ### Hero service `delete` method
 
-block heroes-comp-add
-  :marked
-    The `HeroDetailComponent` does most of the work. All we do is toggle an `*ngIf` flag that
-    swaps it into the DOM when we add a hero and removes it from the DOM when the user is done.
+  The delete HTTP method will be used to delete heroes and its invocation is like to `put` except for the method name:
```

Comment:
This is hard to read and "like to put" isn't quite right. How about:

The hero service's `delete` method` uses the _delete_ HTTP method to remove the hero from the server: